lxd/seccomp/seccomp.go

```diff
@@ -1249,6 +1249,28 @@ func FindTGID(procFd int) (uint32, error) {
 	return 0, fmt.Errorf("Task group leader ID not found")
 }
 
+func isCapableInCtInitUserns(siov *Iovec, cap C.int) (bool, error) {
```

Comment:
Please can you add a doc block comment to this function explaining what it does? Thanks

Reply:
Have done